### PR TITLE
searching api with user input

### LINKED ISSUE #12: Search api with search
call api with the current search state

### PR DESCRIPTION
**Pull request by jmart790**
closes #12

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -1,28 +1,48 @@
...
+const express = require("express");
+const path = require("path");
+const app = express();
 
 // JUST FOR DEMO PURPOSES, PUT YOUR ACTUAL API CODE HERE
-app.get('/home/:query', (request, response) => {
-  axios.get(`https://www.thecocktaildb.com/api/json/v2/${process.env.API_KEY}/filter.php?i=${request.params.query}`)
-    .then(cocktailResponse => response.json(cocktailResponse.data.Search || []))
-})
+app.get("/home/:query", (request, response) => {
```
Comment (Stommy123) [inline server.js]: Would you mind removing this entire route? I think we're substituting it with the one below `/drinks/:item`, they both essentially do the same thing

File: client/src/pages/Home.js [hunk 2]
```diff
@@ -1,14 +1,38 @@
...
+        {/* <div id="drinks"> */}
+        {/* {this.state.map(drink => ( */}
+        {/* // <Link key={movie.imdbID} to={`/movies/${movie.imdbID}`}> */}
+        {/* <div className="drink"> */}
+        {/* <h3>{}</h3> */}
+        {/* <img src={movie.Poster} /> */}
+        {/* </div> */}
+        {/* </Link> */}
+        {/* ))} */}
+        {/* </div> */}
```
Comment (Stommy123) [inline client/src/pages/Home.js]: There seems to be multiple commented out code and `console.log`. These can be helpful while we're debugging or thinking through an issue. But when we're ready to submit a PR, they should be removed as they can hinder other teammates's debugging process

Comment (Stommy123) [reply client/src/pages/Home.js]: Of course if the comments are to clarify what is being done at certain steps, this comment doesn't apply. For example the comments in the `Search` component are perfectly acceptable

**Comment by jmart790** [comment]
Mistake

**Pull request closed without merging**